src/Goerli-DssSpell.t.sol

```diff
@@ -1,4 +1,19 @@
-// SPDX-License-Identifier: AGPL-3.0-or-later
+// SPDX-License-Identifier: GPL-3.0-or-later
```

Comment:
I think this should be AGPL.
We should also fix that on spells-mainnet as well.
```suggestion
// SPDX-License-Identifier: AGPL-3.0-or-later
```